### PR TITLE
add function to get max FAF from faf_expr

### PR DESCRIPTION
**Pull request by KoalaQin**
Similar to what we do to get a popmax/grpmax from 'freq', we will also get a max on faf95 and faf99 and its corresponding pop from the 'faf' field.

### CODE REVIEW

File: gnomad/utils/annotations.py [hunk 1]
```diff
@@ -237,6 +237,72 @@ def faf_expr(
     return faf_expr, hl.eval(faf_meta)
 
 
+def pop_max_for_faf_expr(
```
[the remark this answers is not in the document]
Comment (KoalaQin) [reply gnomad/utils/annotations.py:240]: `gen_anc_grp_max_faf_expr` looks more clear! 
oh, someone has to change L38 `pop_max_expr` too?

File: gnomad/utils/annotations.py [hunk 2]
```diff
@@ -237,6 +237,72 @@ def faf_expr(
...
+            key=lambda f: (-f.faf, f.population),
+        ),
+        lambda fafs: hl.if_else(
+            (hl.len(fafs) > 0) & (fafs[0].faf > 0),
+            hl.struct(faf99_max=fafs[0].faf, faf99_max_pop=fafs[0].population),
+            hl.struct(
+                faf99_max=hl.missing(hl.tfloat), faf99_max_pop=hl.missing(hl.tstr)
+            ),
+        ),
+    )
```
[the remark this answers is not in the document]
Comment (KoalaQin) [reply gnomad/utils/annotations.py:301]: I put in a private function and then annotate the results to have a correct names.

File: gnomad/utils/annotations.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mike-w-wilson) [reply gnomad/utils/annotations.py]: Yeah, that one will be a breaking change so I think we keep it as is for now and do a sweep of the entire repo, moving from pop -> gen_anc everywhere all at once.

**Comment by KoalaQin** [comment]
Do you want to go over it one more time?

**Review by mike-w-wilson — APPROVED** [review]
Just a couple of changes to the language then this LGTM

**Pull request merged**